### DISCUSSION TITLE
Project Status

### DISCUSSION THREAD

**Discussion by william-san** [rosetta-dogecoin/rosetta-dogecoin#63] [General]
Can we get an update on the status of this project? We are considering utilizing the Rosetta to integrate Doge into our data framework. Did this project hit a blocker that makes it not possible, or did it just fall off of the development priorities of the contributors? We may be able to help push a V1 over the line with a little more information on the current status.

**Comment by patricklodder** [comment]
Hi! The main obstruction to me is the cost of developing it further (needs tons of hardware to test mainnet) vs the interest in usage. If there is interest then we can pick this up again.

No real blockers technically.
